index.js

```diff
@@ -8,8 +8,10 @@ var port = process.env.PORT || 3000;
 app.use(express.static(__dirname + '/client'));
 
 app.use(bodyParser.json());
+//parse x-ww-form-urlencoded encoded req bodies
+app.use(bodyParser.urlencoded({extended: true}));
 
-//routes
+//use routes.js
```

Comment:
We might need discuss as a group where the routing will happen. Modularity (separation of concerns) vs consolidation.